package/MDAnalysis/core/AtomGroup.py

```diff
@@ -457,7 +457,84 @@
 # And the return route
 _SINGULAR_PROPERTIES = {v: k for k, v in _PLURAL_PROPERTIES.items()}
 
-_FIFTEEN_DEPRECATION = "This will be removed in version 0.15.0"
+_SIXTEEN_DEPRECATION = "This will be removed in version 0.16.0"
```

Comment:
If this change for the other deprecations is intentional I'm fine with it. 

What I was referring that we currently clean up *a lot* and deprecate old stuff all over. So much that I forget what is supposed to be deprecated when myself. Just a list that we try to keep up to date for us devs to see what deprecations to check for when a release is happening.